test/game_collection_test.rb

```diff
@@ -23,6 +23,28 @@ def test_it_can_create_games_from_csv
     assert_equal 3, @game.home_goals
   end
 
+  def test_it_can_calculate_average_goals_per_game
+    expected = 4.31
+    assert_equal  expected, @game_collection.average_goals_per_game
```

Comment:
why not just assert equal 4.31 instead of expected?

Reply:
good call